srcpkgs/ffmpeg/template

```diff
@@ -26,19 +26,20 @@ makedepends="zlib-devel bzip2-devel freetype-devel alsa-lib-devel libXfixes-deve
  $(vopt_if webp libwebp-devel) $(vopt_if sofa libmysofa-devel)
  $(vopt_if vulkan 'vulkan-loader Vulkan-Headers')
  $(vopt_if drm libdrm-devel) $(vopt_if svtav1 libsvt-av1-devel)
- $(vopt_if srt srt-devel) $(vopt_if rist librist-devel)"
+ $(vopt_if srt srt-devel) $(vopt_if rist librist-devel)
+ $(vopt_if nvdec nv-codec-headers)"
```

Comment:
Please add `configure_args` option to enable/disable nvenc/nvdec on build_options, let's say someone want to enable nvdec and disable nvenc for whatever reason.

Reply:
It would be wise to put nvdec and nvenc near each other

Reply:
Hey @sgn,

I've made the changes as requested.
I've also refactored `makedepends=` so the entries are in the correct text width.